bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -12,6 +12,8 @@ spec:
       app: linuxptp-daemon
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 33%
```

Comment:
why you use 33% and not 1 for example any special reason?

Reply:
1 makes upgrades very slow... 10% is what most things do, but components choose something larger.